src/test/java/com/meilisearch/sdk/SearchRequestTest.java

```diff
@@ -0,0 +1,99 @@
+package com.meilisearch.sdk;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class SearchRequestTest {
+
+    @Test
+    public String toString() {
```

Comment:
Can you create different tests to increase the readability? It is very hard to understand it :/